H5P/laravel-h5p/src/LaravelH5p/Http/Controllers/AjaxController.php

```diff
@@ -15,6 +15,10 @@ class AjaxController extends Controller
 {
     public function libraries(Request $request)
     {
+        // headers for CORS
+        header('Access-Control-Allow-Origin: *');
+        header('Access-Control-Allow-Methods: POST, GET, OPTIONS');
+        
```

Comment:
Why do we enable CORS in this way?

Reply:
Laravel cors are not being picked in H5P. If you have better way you can inspect

Reply:
I will merge it for now. I don't have time to spend on this now.

Reply:
Our current configurations of fruitcake/laravel-cors are not being in-effect.